src/source/tealiumCore.brs

```diff
@@ -206,6 +245,22 @@ function TealiumCore(logLevel as Integer, account as String, profile as String,
                 return vid
             end function
 
+            'Gets app UUID
+            '@return: String 32 character long alphanumeric
+            _GetAppUUID: function () as String
```

Comment:
could we extract a common method with a constant parameter for both visitorId and appUUID?